x/paramfilter/test/governance_params_test.go

```diff
@@ -0,0 +1,549 @@
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/x/paramfilter"
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	testutil "github.com/celestiaorg/celestia-app/test/util"
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	bsmoduletypes "github.com/celestiaorg/celestia-app/x/blobstream/types"
+	minttypes "github.com/celestiaorg/celestia-app/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	params "github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctransfertypes "github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
+	ibcclienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
+	ibcconnectiontypes "github.com/cosmos/ibc-go/v6/modules/core/03-connection/types"
+)
+
+type TestSuite struct {
+	suite.Suite
+
+	app        *app.App
+	ctx        sdk.Context
+	govHandler v1beta1.Handler
+	pph        paramfilter.ParamBlockList
+}
+
+func (suite *TestSuite) SetupTest() {
+	suite.app, _ = testutil.SetupTestAppWithGenesisValSet(app.DefaultConsensusParams())
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.govHandler = params.NewParamChangeProposalHandler(suite.app.ParamsKeeper)
+	suite.pph = paramfilter.NewParamBlockList([2]string{})
+
+	minter := minttypes.DefaultMinter()
+	suite.app.MintKeeper.SetMinter(suite.ctx, minter)
+}
+
+func TestTestSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping handler test suite in short mode.")
```

Comment:
[nit] can the name in the skip message match the name of the test suite which matches the name of the file? Currently they're all different so it may be difficult for a developer unfamiliar with this code to find this test based on the skip message. Proposal to use:

- TestSuite => GovernanceParamsTestSuite
- TestTestSuite => TestGovernanceParamsTestSuite
- "skipping handler test suite in short mode." => "skipping governance params test suite in short mode."